content/en/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -12,9 +12,9 @@ This page explains how Kubernetes objects are represented in the Kubernetes API,
 {{% /capture %}}
 
 {{% capture body %}}
-## Understanding Kubernetes Objects
+## Understanding Kubernetes objects {#kubernetes-objects}
```

Comment:
This is the first heading anchor, so if any incoming links break, the reader experience isn't really affected.